### PR TITLE
feat: add ui5 middleware code coverage

### PR DESCRIPTION
**Pull request by flovogt**
Allows client side code coverage generation for ES6+ code powered by https://github.com/SAP/ui5-tooling-extensions/tree/main/packages/middleware-code-coverage.

Projects generated for UI5 versions before 1.113.0 will stick to old "Blanket.js" based code coverage determination limited to ES5 language specification.

JIRA: CPOUI5FOUNDATION-721

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -131,6 +131,14 @@ export default class extends Generator {
 			} else {
 				this.config.set("defaultTheme", "sap_fiori_3");
 			}
+
+			// set qunit coverage file
+			if (semver.gte(props.frameworkVersion, "1.113.0")) {
+				// set qunit-coverage file
```
Comment (matz3) [inline generators/app/index.js:137]: duplicated comment. Better add a reference to the middleware README which contains this minimum version requirement.

File: generators/app/templates/webapp/test/unit/unitTests.qunit.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
 		<link rel="stylesheet" type="text/css" href="../../resources/sap/ui/thirdparty/qunit-2.css" />
 		<script src="../../resources/sap/ui/thirdparty/qunit-2.js"></script>
 		<script src="../../resources/sap/ui/qunit/qunit-junit.js"></script>
-		<script src="../../resources/sap/ui/qunit/qunit-coverage.js"></script>
+		<script src="../../resources/sap/ui/qunit/<%= qunit-coverage-file %>"></script>
```
Comment (matz3) [inline generators/app/templates/webapp/test/unit/unitTests.qunit.html:22]: `Error: qunit is not defined`
I think the variables must be valid JavaScript variables (not containing dashes). Existing variables use camelCase.

File: generators/app/index.js (hunk 1 above)
Comment (flovogt) [reply generators/app/index.js]: done

File: generators/app/templates/webapp/test/unit/unitTests.qunit.html (hunk 2 above)
Comment (flovogt) [reply generators/app/templates/webapp/test/unit/unitTests.qunit.html]: Thx. Done.

**Review by matz3 — APPROVED** [review]

**Pull request merged**